helm/solr/Chart.yaml

```diff
@@ -42,15 +42,12 @@ annotations:
   # Allowed syntax is described at: https://artifacthub.io/docs/topics/annotations/helm/#example
   artifacthub.io/changes: |
     - kind: added
-      description: Addition 1
+      description: SolrClouds now support namespace sharing among pod containers in a pod.
```

Comment:
Every time I come back to this project after a lull, I have trouble deciding which chart should hold a given change-entry.

Hopefully I've picked correctly here - I picked the 'solr' chart because the newly-exposed option affects the Solr pods (and not the operator pod itself)

Reply:
Solr Operator, since that is the main chart (you can use the solr operator chart without using the solr chart, but not really the other way around). The Solr chart changeling is only used for changes relating to that helm chart.